### PR TITLE
Governance refactor

### PR DESCRIPTION
**Pull request by HarshCasper**
## Description

This PR contains: 

- Refactor of `What is a Project?` docs 
- Refactor of `Text Collaboration` docs
- Refactor of `Continuous Improvements` docs
- Refactor of `Coding Guideline` docs
- Refactor of `Strategic Plans` docs
- Addition of a README.md to `governance` sub-directory 

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [X] I have checked my code and corrected any misspellings
- [ ] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [X] I have signed the commit message to agree to the Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

File: Governance/STRATEGIC-PLANS.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+## The Annual Strategic Plans
+
+-   [2019](./Strategic-Plan/2019.md)
+-   [2018](./Strategic-Plan/2018.md)
+-   [2017](./Strategic-Plan/2017.md)
```
Comment (aornugent) [inline Governance/STRATEGIC-PLANS.md:5]: Please file an issue to add the 2020 and 2021 plans, and I'll dig them out!

**Comment by aornugent** [comment]
Fantastic @HarshCasper - merge when ready

**Comment by HarshCasper** [comment]
Hi @aornugent @Sagar2366 

It would be great if you can review this PR. 

A few pointers that I would like to highlight on top of it: 

- Files that were empty and had `To be completed` parts were deleted. I will make separate issues for new contributors to tackle up.
- Do we want to convert those PDF files to Markdown for easy collaboration and editing. Or just leave them as it is for their place.
- Coding Style Guidelines contains information specific to C++. Should we add Python and JS as well, because newer projects concerns around these areas? 

Thanks a lot for helping with all these!

File: Governance/STRATEGIC-PLANS.md (hunk 1 above)
Comment (HarshCasper) [reply Governance/STRATEGIC-PLANS.md]: It was there in the `Strategic Plan` subdirectory. I have fixed them up.

**Pull request merged**